src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -909,24 +909,24 @@ inline RppStatus rpp_store24_f32pln3_to_f32pkd3_avx(Rpp32f *dstPtr, __m256 *p)
     return RPP_SUCCESS;
 }
 
-inline RppStatus rpp_load8_f32_to_f32_avx(Rpp32f *srcPtr, __m256 *p)
+inline RppStatus rpp_load16_f32_to_f32_avx(Rpp32f *srcPtr, __m256 *p)
 {
     p[0] = _mm256_loadu_ps(srcPtr);
+    p[1] = _mm256_loadu_ps(&srcPtr[8]);
```

Comment:
please change it to use _mm256_loadu_ps(srcPtr+8)

Reply:
Done